webapp/client/web_client.jsx

```diff
@@ -14,6 +17,7 @@ class WebClientClass extends Client {
     constructor() {
         super();
         this.enableLogErrorsToConsole(true);
+        this.connected = true;
```

Comment:
Oh, that makes sense. Maybe just rename it to `internetConnected` or `hasInternetConnection` to make it clearer?